File: java/8.10/example3.java
import java.util.Scanner;

public class example3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter 1 or 2");
        Integer choice = Integer.parseInt(scanner.nextLine());
        if (choice == 1) {
            System.out.println("Enter a word");
            String word = scanner.nextLine();
            for (Integer count = 0; count < word.length(); count++) {
                String character = word.substring(count, count+1);
                if (character.equals("a") || character.equals("e") || character.equals("i") || character.equals("o") || character.equals("u")) {
                    System.out.println(character);
                }
            }
        }
    }
}

File: python/8.10/example3.py
choice = int(input("Enter 1 or 2")) # Setting the value to the interger of the value inputed
if choice == 1: # Checking if choice is equal to 1
    word = input("Enter a word") # Setting the value to the value inputed
    for count in range(0, len(word)): # Runnig the loop for as long as the length of word
        character = word[count:count + 1] # Grabbing the curent charictor
        if character == "a" or character == "e" or character == "i" or character == "o" or character == "u": # Checking if the current charictor is a voel
            print(character) # Printing